File: Week 1/mario-more/mario.c
// Recreates the pyramid of blocks from Super Mario Brothers
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Gets the height from the user
    int height;
    do
    {
        height = get_int("Height : ");
    }
    while (height < 1 || height > 8);

    // Creates the pyramid
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < (height - i); j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6/sentimental-mario-more/mario.py
# Recreates the pyramid of blocks from Super Mario Brothers

from cs50 import get_int

# Get the height of the pyramid as an input from the user
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(1, height + 1):
    print((" " * (height - i)) + ("#" * i) + "  " + ("#" * i))
